src/app/pages/AccountPage/index.tsx

```diff
@@ -91,7 +92,7 @@ export function AccountPage(props: Props) {
 
   // Reload account balances if address or network changes
   useEffect(() => {
-    dispatch(accountActions.fetchAccount(address))
+    dispatch(accountActions.fetchAccount({address,type:walletType}))
```

Comment:
This is wrong: address comes from URL

Reply:
How to reproduce: Use Search box and search for address.

Reply:
done